src/control/server/storage_nvme_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	. "github.com/daos-stack/daos/src/control/common/storage"
 	. "github.com/daos-stack/daos/src/control/lib/spdk"
 	"github.com/daos-stack/daos/src/control/logging"
+	"github.com/google/go-cmp/cmp"
```

Comment:
Looks like these imports got auto-added to the wrong place.

Reply:
okay if I fix in the next PR (have a follow-on for this area)